.github/workflows/kind.yaml

```diff
@@ -42,6 +43,8 @@ jobs:
       - name: Collect all logs
         if: always()
         run: |
+          # Just dump the secrets to debug later
+          kubectl get secrets -n flotta  -o json
```

Comment:
sorry for the pickiness :-P but there are two spaces after `flotta` XD
Also, is json more convenient than yaml for debug? or just personal preference?

Reply:
Json because is easy to write the certs to local using JQ